app/assets/javascripts/home.coffee

```diff
@@ -31,7 +31,14 @@ set_coords = (position) ->
         $('.test-speed-btn').prop('disabled', false)
         $('.location-warning').addClass('hide')
         $.getJSON 'https://jsonip.com/?callback=?', (result_data) ->
```

Comment:
Remove `?callback=?` and you should be able to also remove `.split(',')[0]` below.